zmarkdown/config.js

```diff
@@ -0,0 +1,322 @@
+        ['www.', ''],
+        ['vimeo.com/', 'player.vimeo.com/video/'],
+      ]
+    },
+    'vimeo.com': {
+      tag: 'iframe',
+      width: 500,
+      height: 281,
+      disabled: false,
+      replace: [
+        ['http://', 'https://'],
+        ['www.', ''],
+        ['vimeo.com/', 'player.vimeo.com/video/'],
+      ]
+    },
+    'www.youtube.com': {
+      tag: 'iframe',
+      width: 560,
+      height: 315,
+      disabled: false,
+      replace: [
+        ['watch?v=', 'embed/'],
+        ['http://', 'https://'],
+      ],
+      removeAfter: '&'
+    },
+    'youtube.com': {
+      tag: 'iframe',
+      width: 560,
+      height: 315,
+      disabled: false,
+      replace: [
+        ['watch?v=', 'embed/'],
+        ['http://', 'https://'],
+      ],
+      removeAfter: '&'
+    },
+    'youtu.be': {
+      tag: 'iframe',
+      width: 560,
+      height: 315,
+      disabled: false,
+      replace: [
+        ['watch?v=', 'embed/'],
+        ['youtu.be', 'www.youtube.com/embed'],
+      ],
+      removeAfter: '&'
+    },
+    'screen.yahoo.com': {
+      tag: 'iframe',
+      width: 624,
+      height: 351,
+      disabled: false,
+      append: '?format=embed&player_autoplay=false'
+    },
+    'www.ina.fr': {
+      tag: 'iframe',
+      width: 620,
+      height: 349,
+      disabled: false,
+      replace: [
+        ['www.', 'player.'],
+        ['/video/', '/player/embed/'],
+      ],
+      append: '/1/1b0bd203fbcd702f9bc9b10ac3d0fc21/560/315/1/148db8',
+      removeFileName: true
+    },
+    'www.jsfiddle.net': {
+      tag: 'iframe',
+      width: 560,
+      height: 560,
+      disabled: false,
+      replace: [
+        ['http://', 'https://'],
+      ],
+      append: 'embedded/result,js,html,css/',
+      match: /https?:\/\/(www\.)?jsfiddle\.net\/([\w\d]+\/[\w\d]+\/\d+\/?|[\w\d]+\/\d+\/?|[\w\d]+\/?)$/,
+    },
+    'jsfiddle.net': {
+      tag: 'iframe',
+      width: 560,
+      height: 560,
+      disabled: false,
+      replace: [
+        ['http://', 'https://'],
+      ],
+      append: 'embedded/result,js,html,css/',
+      match: /https?:\/\/(www\.)?jsfiddle\.net\/([\w\d]+\/[\w\d]+\/\d+\/?|[\w\d]+\/\d+\/?|[\w\d]+\/?)$/,
+    }
+  },
+
+  captions: {
+    external: {
+      table: 'Table:',
+      gridTable: 'Table:',
+      code: 'Code:',
+      math: 'Equation:',
+    },
+    internal: {
+      iframe: 'Video:',
+      math: 'Equation:',
+      inlineMath: 'Equation:'
+    }
+  },
+
+  ping: {
+    pingUsername: (username) => {
+      return false
+      // try {
+      //   const result = request(
+      //     'HEAD',
+      //     `https://zestedesavoir.com/api/membres/exists/?search=${username}`,
+      //     {timeout: 300}
+      //   )
+      //   return result.statusCode === 200
+      // } catch (ex) {
+      //   console.error(ex)
+      //   return false
+      // }
+    },
+    userURL: (username) => `/membres/voir/${username}/`,
+  },
+}
+
+const rebberConfig = {
```

Comment:
remember we need to override `image` and `figure.image` to use the custome `\\image` command in rebber.